docs/telemetry.rst

```diff
@@ -226,4 +226,20 @@ Example of recorded documents given two data streams in the cluster::
 
 Supported telemetry parameters:
 
-* ``data-stream-stats-sample-interval`` (default 10): A positive number greater than zero denoting the sampling interval in seconds.
\ No newline at end of file
+* ``data-stream-stats-sample-interval`` (default 10): A positive number greater than zero denoting the sampling interval in seconds.
+
+
+master-node-stats
+-----------------
+
+The master-node-stats telemetry device regularly reports the name of the master node by using:
+
+ 1. the `Cluster State API <https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-state.html>`_, specifically `/_cluster/state/master_node` to determine the id of the master node
```

Comment:
Nit: The docs here are outdated now since we just use the cat client. 

Though, I also wonder if we should even expose this in the docs given we intend for it to be an internal telemetry device? @danielmitterdorfer, thoughts?

Reply:
I agree, we should skip that for internal telemetry devices.

Reply:
Thanks, I forgot to remove it when I updated the CLI output at the top of the page. Nice catch!

I removed it now.